### ISSUE TITLE
Lock screen widget rotation bug

### ISSUE THREAD

**Issue by PadeLee** [Matchstic/Xen-HTML#17]
Whenever I rotate my screen in an app or at the home screen and lock my screen while having it horizontal, it bugs the lock screen widget.

**Comment by Matchstic** [comment]
Are you running on a device that is capable of rotating at the Lockscreen?

**Comment by PadeLee** [comment]
No, i am running i6 Plus resolution on my i6 using a custom IOGraphicsFamily.plist. Everything is normal my HS rotates but my LS does not. 

Thanks for replying btw :)

**Comment by Matchstic** [comment]
Any chance you can link me some instructions to get the same setup? I also have an i6, so would be able to replicate your settings as a result for testing.

**Comment by PadeLee** [comment]
[cid:F372B9F2-D613-4A9A-A2FC-079AB6FE1308]

Copy paste the file to this directory: /private/var/mobile/Library/Preferences

Reboot. That's what I am using no Upscale no LittleBrother. I don't know what else to provide you with.  I also use a lock screen widget part of the E8.WdgtPack.8 but I don't think that matters as I have tested many widgets and the bug applies to all of them.

Here's how to reproduce the bug, after you apply the new resolution go to your home screen or any app rotate the display and press the sleep button. When you wake up the phone the widget will be fuked up.

Best of luck in resolving that, you are such a good dev keep up:)

Any chance you can link me some instructions to get the same setup? I also have an i6, so would be able to replicate your settings as a result for testing.

**Comment by PadeLee** [comment]
Just saw your email :$$ that was fast good job man

Scratch that, issue resolved for next update.
